components/count-badge/count-badge-mixin.js

```diff
@@ -165,7 +166,10 @@ export const CountBadgeMixin = superclass => class extends LocalizeCoreElement(R
 		};
 		if (this.maxDigits && this.number.toString().length > this.maxDigits) {
 			numberString = `${'9'.repeat(this.maxDigits)}`;
+			numberString = formatNumber(numberString);
```

Comment:
`formatNumber` is expecting a JavaScript `Number` here... so maybe put it through `parseInt` first just to be safe?